code/controllers/subsystem/timer.dm

```diff
@@ -71,72 +71,79 @@ SUBSYSTEM_DEF(timer)
 		for(var/I in second_queue)
 			log_world(get_timer_debug_string(I))
 
+	var/cut_start_index = 1
 	var/next_clienttime_timer_index = 0
 	var/len = length(clienttime_timers)
 
-	for(next_clienttime_timer_index in 1 to len)
-		if(MC_TICK_CHECK)
+	for (next_clienttime_timer_index in 1 to len)
+		if (MC_TICK_CHECK)
```

Comment:
stop sporadically changing these spaces.... you aren't even doing them everywhere.

Reply:
The way I do things, language statements (if, for, while, ...) I prefer a space, maybe because they are a word to me. For user defined procs I don't use a space. At least that's the last way I remember writing any form of code

Reply:
Paradise contributing guidelines dictate that there is no space. It's not wrong, just out standards are different.